openssl-dynamic/src/main/c/ssl_private.h

```diff
@@ -64,6 +64,7 @@
 #include <openssl/rand.h>
 #include <openssl/x509v3.h>
 #include <openssl/hmac.h>
+#include <openssl/dh.h>
```

Comment:
This is needed now as in newer version of OpenSSL this is required (and on older version it cant harm).